google-cloud-logging/lib/google/cloud/logging/v2/doc/google/api/metric.rb

```diff
@@ -122,7 +123,27 @@ module Api
     #     Use sentence case without an ending period, for example "Request count".
     #     This field is optional but it is recommended to be set for any metrics
     #     associated with user-visible concepts, such as Quota.
+    # @!attribute [rw] metadata
+    #   @return [Google::Api::MetricDescriptor::MetricDescriptorMetadata]
+    #     Optional. Metadata which can be used to guide usage of the metric.
     class MetricDescriptor
+      # Additional annotations that can be used to guide the usage of a metric.
+      # @!attribute [rw] launch_stage
+      #   @return [Google::Api::LaunchStage]
+      #     The launch stage of the metric definition.
+      # @!attribute [rw] sample_period
+      #   @return [Google::Protobuf::Duration]
+      #     The sampling period of metric data points. For metrics which are written
+      #     periodically, consecutive data points are stored at this time interval,
+      #     excluding data loss due to errors. Metrics with a higher granularity have
+      #     a smaller sampling period.
+      # @!attribute [rw] ingest_delay
+      #   @return [Google::Protobuf::Duration]
+      #     The delay of data points caused by ingestion. Data points older than this
+      #     age are guaranteed to be ingested and available to be read, excluding
+      #     data loss due to errors.
+      class MetricDescriptorMetadata; end
```

Comment:
@dazuma Should this data be exposed in the Ruby API?

Reply:
Good question. Let's not for now, but open a cleanup issue to get some clarification from the stackdriver team.

Reply:
Okay, the issue created for this is #2463.